node/config/def.go

```diff
@@ -107,6 +107,9 @@ func DefaultFullNode() *FullNode {
 			MaxFilterResults:        10000,
 			MaxFilterHeightRange:    2880, // conservative limit of one day
 		},
+		Fevm: FevmConfig{
+			EnableEthHashToFilecoinCidMapping: false,
```

Comment:
nit: why this value but not `EthTxHashMappingLifetimeDays`

Reply:
@geoff-vball ^